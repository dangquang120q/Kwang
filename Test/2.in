6
7
6213 9815 2924 9904 9051 1468 697 
15
1885 6909 1887 4051 3988 9784 8577 8650 8375 6299 3279 1805 9915 1629 1109 
15
5611 9114 8895 2759 1507 9851 3563 1595 469 3636 1061 1740 2718 2229 3154 
11
341 4000 1551 2280 3143 6344 7525 7266 4200 2844 6542 
63
9889 5677 6105 938 7624 6706 7743 5366 7964 3506 8352 8732 117 8015 6681 7389 7851 7320 9017 7675 2738 2745 3547 7422 9018 5291 9862 7334 9706 7400 5141 7281 4258 4503 6883 7823 3887 5114 4919 1928 6128 9628 4419 3014 7739 5217 7275 805 7410 464 2921 41 4353 4585 5300 3468 206 3684 6978 7286 6409 8779 3031 
74
9569 9413 3842 9760 3093 3795 1460 4113 5858 6335 3321 6317 7321 4274 2946 4309 5142 856 4763 8518 5531 2165 3617 3237 4959 1214 3969 6426 4459 4284 5454 600 3374 2933 1566 7351 8790 7809 3942 4883 1990 7535 4303 3394 9191 1443 7529 9912 3113 9801 8443 4845 4312 9376 9124 7498 1379 3048 6536 3630 2233 4485 6059 9391 4367 6426 920 6829 8858 6331 6206 2861 9288 5763 
