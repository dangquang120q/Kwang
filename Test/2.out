28770
28349
27860
21333
29457
29473
